echopype/convert/set_groups_azfp.py

```diff
@@ -382,6 +408,17 @@ def set_beam(self) -> List[xr.Dataset]:
                         "valid_min": 0.0,
                     },
                 ),
+                "transmit_type": (
+                    [],
+                    "CW",
+                    {
+                        "long_name": "Type of transmitted pulse",
+                        "flag_values": ["CW"],
+                        "flag_meanings": [
+                            "Continuous Wave",
```

Comment:
I've been mulling over about this. It is true that CW is Continuous Wave, but it is an approximation. The actual transmit signal is gated sine wave. I wonder if this would be misleading. This is something defined in the convention, but do you think we can add something to make it "Continuous Wave (gated sine wave)"?